styles.css

```
@@ -1,6 +1,40 @@
 @charset 'UTF-8';
 
-h1 {
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+nav {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 2.4rem;
+  font-size: 1.2rem;
   font-family: Arial, Helvetica, sans-serif;
```

```suggestion
  font-family: 'Work Sans';
```
O nosso template utiliza a fonte 'Work Sans'. Voce poderia trocar por gentileza? Nao sera apenas uma questao de mudar o nome. Da uma olhada nesse exemplo. Acho que vai te ajudar https://www.w3schools.com/howto/tryit.asp?font=Work%20Sans

---

Sim. Sem problemas!
Eu vou deixar para commitar na nossa reunião para o pessoal ver que é possível aceitar os seus commits no meu PR.
Como você identificou a fonte?
A ferramenta que eu tentei utilizar para identificar não funcionou.

---

Alterado!
Obrigado, @jhrodriguesit !